src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.Managed.cs

```diff
@@ -344,15 +360,30 @@ void IDirectManipulationHandler.OnInertiaStarting(GestureRecognizer recognizer,
 				return;
 			}
 
-			// As we run animation by our own, we request to have pretty long delay between ticks to avoid too many updates.
-			args.Manipulation.Inertia!.Interval = TimeSpan.FromMilliseconds(100);
+			var inertia = args.Manipulation.Inertia ?? throw new InvalidOperationException("Inertia processor is not available.");
+			if (OperatingSystem.IsIOS())
```

Comment:
Non Android and non iOS targets will use some default behavior here?

Reply:
The windows/winUI behavior yes. Planning to also change that behavior for wasm-iOS and wasm-Android, but we don't yet have flags for that. I want to validate the new behavior in real apps first.